srcpkgs/autorunner/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'autorunner'
+pkgname=autorunner
+version=1.0.0
+revision=1
+conf_files="/etc/autorunner.config"
+depends="bash procps-ng coreutils grep gawk"
```

Comment:
they are all part of base-system and even base-voidstrap

Reply:
@teldra you can omit these dependencies, as they are part of `base-system` (and supposed to be already installed...)